cosmo_tester/test_suites/ha/ha_cluster_scenarios_test.py

```diff
@@ -71,31 +72,40 @@ def cluster(
 
 
 @pytest.fixture(scope='function')
-def hello_world(cfy, cluster, attributes, ssh_key, tmpdir, logger):
-    hw = HelloWorldExample(
-        cfy, cluster.managers[0], attributes, ssh_key, logger, tmpdir)
-    hw.blueprint_file = 'openstack-blueprint.yaml'
-    hw.inputs.update({
-        'agent_user': attributes.centos7_username,
-        'image': attributes.centos7_image_name,
-    })
-
-    yield hw
-    if hw.cleanup_required:
-        logger.info('Hello world cleanup required..')
-        cluster.managers[0].use()
-        hw.cleanup()
-
-
-def test_data_replication(cfy, cluster, hello_world,
+def hello_worlds(cfy, cluster, attributes, ssh_key, tmpdir, logger):
```

Comment:
I've already seen a `hello_worlds` fixture. Maybe we can differentiate this one by calling it something else?

Reply:
Yeah, I think maybe a little differentiation at least to make it clear which one is being viewed if someone has been wandering in a maze of twisty lines of code, all alike, for hours.